### PR TITLE
feat: rewrite introductory sections

### PR DESCRIPTION
**Pull request by lthoerner**
I'm working on rewording/clarifying existing intro text and writing new stuff. This is a work in progress.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,23 +3,19 @@
...
 This is the documentation project for [Ratatui](https://github.com/tui-rs-revival/ratatui), a Rust
-library to build rich terminal user interfaces and dashboards. The documentation can be read at
-<https://tui-rs-revival.github.io/ratatui-book/>.
+library to build rich terminal user interfaces and dashboards. Note that this documentation is supplementary
+to the technical documentation, and we also maintain crate documentation. This is more of a guide book
+to help our users understand how to build a TUI with Ratatui. If you are looking for documentation on
+structs, enums, traits, methods, etc., please visit our [crates.io documentation](https://docs.rs/ratatui/0.21.0/ratatui/).
 
 [![License](https://img.shields.io/crates/l/ratatui?style=for-the-badge)](./LICENSE)
-[![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/tui-rs-revival/ratatui-book/mdbook.yml?label=Github%20Pages%20Deployment&logo=github&style=for-the-badge)](https://github.com/tui-rs-revival/ratatui-book/actions/workflows/mdbook.yml)
```
Comment (joshka) [inline README.md:10]: Why drop the badges?

File: README.md [hunk 2]
```diff
@@ -3,23 +3,19 @@
...
 
-## Contributing
-
-The process for writing these docs is just getting started. Feel free to give some suggestions via
-[GitHub Discussions](https://github.com/tui-rs-revival/ratatui-book/discussions) or chat with us on
-[Discord](https://discord.com/channels/1070692720437383208/1115053951000268832).
+## Viewing the Book
 
-## Installation
+To read the book in your browser, click [here](https://tui-rs-revival.github.io/ratatui-book/).
```
Comment (joshka) [inline README.md]: Can you phrase this to link some other word than “here”? even a bare url would be a improvement.

File: README.md (hunk 1 above)
Comment (lthoerner) [reply README.md:10]: I do not feel that workflow status is at all relevant for people reading documentation.

File: README.md (hunk 2 above)
Comment (lthoerner) [reply README.md]: Why is the link text an issue? Is this due to the fact that some people will be viewing this in a context where hyperlinks are not supported?

**Comment by lthoerner** [comment]
As requested in #7, I've moved my "Choosing a Backend" section to its own dedicated file.

File: README.md [hunk 3]
```diff
@@ -3,23 +3,19 @@
 <img align="left" src="https://avatars.githubusercontent.com/u/125200832?s=128&v=4" alt="ratatui logo">
 
 This is the documentation project for [Ratatui](https://github.com/tui-rs-revival/ratatui), a Rust
-library to build rich terminal user interfaces and dashboards. The documentation can be read at
-<https://tui-rs-revival.github.io/ratatui-book/>.
+library to build rich terminal user interfaces and dashboards. Note that this documentation is supplementary
+to the technical documentation, and we also maintain crate documentation. This is more of a guide book
```
Comment (mindoodoo) [inline README.md]: Isn't the technical documentation the same as the crate documentation ?

File: README.md [hunk 4]
```diff
@@ -3,23 +3,19 @@
...
+to help our users understand how to build a TUI with Ratatui. If you are looking for documentation on
+structs, enums, traits, methods, etc., please visit our [crates.io documentation](https://docs.rs/ratatui/0.21.0/ratatui/).
```
Comment (mindoodoo) [inline README.md]: > structs, enums, traits, methods, etc.

I would skip this listing in favour of something like "technical documentation" or "crate documentation" as you used earlier.

> please visit our  [crates.io documentation](https://docs.rs/ratatui/0.21.0/ratatui/)

Crate documentation ? or maybe docs.rs documentation ? crates.io documentation sounds a bit wrong to me, what do you think ? Of course I'm nitpicking here.

File: src/getting-started/choosing-a-backend.md [hunk 5]
```diff
@@ -1 +1,26 @@
...
+1. Do you need your program to Windows-compatible?
+    - If yes, use Crossterm or Termwiz.
+    - If no, use Crossterm or Termion.
+2. Do you already have experience with one of the backends?
+    - If yes, consider using the backend you are most familiar with, as long as it meets your platform
+    compatibility needs.
+    - If no, consider using Crossterm.
+3. Do you intend for the TUI to be used primarily with the Wezterm terminal emulator?
+    - If yes, use Termwiz.
+    - If no, use Crossterm or Termion.
```
Comment (mindoodoo) [inline src/getting-started/choosing-a-backend.md:22]: Food for thought, might be interesting to offer some type of flow chart here. What are your thoughts on that option ?

File: src/getting-started/choosing-a-backend.md
Comment (mindoodoo) [inline src/getting-started/choosing-a-backend.md:1]: Maybe make the backends pop out a bit more by putting them in bold ?

**Review by mindoodoo — COMMENTED** [review]
Tons of little nitpicks and suggestions, otherwise looks good :)

File: README.md (hunk 3 above)
Comment (lthoerner) [reply README.md]: They are the same, it's just poorly-worded. I meant "the technical documentation is at crates.io" basically.

File: src/getting-started/choosing-a-backend.md (hunk 5 above)
Comment (lthoerner) [reply src/getting-started/choosing-a-backend.md:22]: I don't think it's straightforward enough to have a flowchart. I mostly want to get across the idea that people should be using Crossterm unless they have a compelling reason not to.

**Comment by kdheepak** [comment]
Hi @lthoerner I'm incorporating your changes from this PR (cherry-picking) and making more edits as I overhaul the book. Would it be okay for you if we closed this PR?

**Comment by lthoerner** [comment]
> Hi @lthoerner I'm incorporating your changes from this PR (cherry-picking) and making more edits as I overhaul the book. Would it be okay for you if we closed this PR?

Yes

**Comment by kdheepak** [comment]
I've included the changes from this PR. I'll update them based on some of these comments as I'm updating the rest of the documentation.

**Pull request closed without merging**